File: src/backend/studio-reservation/src/main/java/org/taonaw/studio_reservation/usecase/command/member/registerMemberAccount/RegisterMemberAccountCommand.java
package org.taonaw.studio_reservation.usecase.command.member.registerMemberAccount;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import org.taonaw.studio_reservation.domain.model.shared.*;

@Getter
@Builder
@AllArgsConstructor
public class RegisterMemberAccountCommand {
    private PersonName name;
    private EmailAddress emailAddress;
    private PlainTextPassword plainTextPassword;
    private DateOfBirth dateOfBirth;
    private PhoneNumber phoneNumber;
}

File: src/backend/identityaccess/src/main/kotlin/org/taonaw/identityaccess/usecase/memberAccount/registerMemberAccount/RegisterMemberAccountCommand.kt
package org.taonaw.identityaccess.usecase.memberAccount.registerMemberAccount

import org.taonaw.identityaccess.domain.model.memberAccount.ContractInformation
import org.taonaw.identityaccess.domain.model.memberAccount.MemberName
import org.taonaw.identityaccess.domain.model.shared.PlainTextPassword

data class RegisterMemberAccountCommand(
        val memberName: MemberName,
        val password: PlainTextPassword,
        val contractInformation: ContractInformation) {
}